test/core.jl

```diff
@@ -4848,12 +4856,15 @@ end
 let ni128 = sizeof(FP128test) ÷ sizeof(Int),
     ns128 = sizeof(FP128align) ÷ sizeof(Int),
     nbit = sizeof(Int) * 8,
-    arr = reinterpret(FP128align, collect(Int, 1:(2 * ns128))),
+    arr = Vector{FP128align}(2),
```

Comment:
makes sense – we want to allocate this with the larger alignment (FP128align), but fill it with the smaller alignment (Int)